MyModTest/RandomGameTest.cs

```diff
@@ -610,115 +684,114 @@ protected int GetRandomNumber()
         }
 
         // Example test implementations follow
-/*
-        [Test]
-        public void TestRandomGameToCompletion()
-        {
-            //for (int i = 0; i < 200; i++)
-            {
-                GameController gameController = SetupRandomGameController(false);
-                RunGame(gameController);
-            }
-        }
-
-        [Test]
-        public void TestSomewhatReasonableGameToCompletion()
-        {
-            GameController gameController = SetupRandomGameController(true);
-            RunGame(gameController);
-        }
-
-        [Test]
-        public void TestSkyScraper()
-        {
-            //for (int i = 0; i < 10; i++)
-            {
-                GameController gameController = SetupRandomGameController(false, useHeroes: new List<string> { "SkyScraper" });
-                RunGame(gameController);
-            }
-        }
-
-        [Test]
-        public void TestTachyon_SupersonicResponse()
-        {
-            //for (int i = 0; i < 1000; i++)
-            {
-                SetupRandomGameController(true, useHeroes: new List<string> { "Tachyon" });
-                PreferredCardsToPlay = new string[] { "SupersonicResponse", "FleetOfFoot", "HUDGoggles", "PushingTheLimits" };
-                RunGame(this.GameController);
-            }
-        }
-
-        [Test]
-        public void TestCelestialTribunal()
-        {
-            //for (int i = 0; i < 10; i++)
-            {
-                GameController gameController = SetupRandomGameController(true, overrideEnvironment:"TheCelestialTribunal");
-                RunGame(gameController);
-            }
-        }
+        /*
```

Comment:
Ugh, I didn't see the whitespace getting mangled on this file. I'll revert these changes.